src/main/java/edu/wright/airviewer2/AIRViewerController.java

```diff
@@ -92,7 +102,13 @@ private AIRViewerModel promptLoadModel(String startPath) {
             File file = fileChooser.showOpenDialog(stage);
             if (null != file) {
                 String path = file.getCanonicalPath();
+                while(!path.endsWith(".pdf")) {
```

Comment:
Some of the edited code is left uncommented. Could you please check and make the necessary changes?

Reply:
Given a comment on the requested modified code